lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -302,16 +312,22 @@ def write(chunk)
             entry['metadata']['severity'] = 'DEFAULT'
           end
 
+          # TODO(a-robinson): Consider falling back to parsing the namespace,
+          # pod, and container names out of the log name if we don't have this
+          # information, which depends on the kubernetes_metadata_filter
+          # filter plugin.
+          if @service_name == CONTAINER_SERVICE && record['kubernetes']
+            handle_container_metadata(record, entry)
+          end
+
           # If a field is present in the label_map, send its value as a label
           # (mapping the field name to label name as specified in the config)
           # and do not send that field as part of the payload.
-          unless label_map.nil?
+          if @label_map
             labels = {}
+            labels = entry['metadata']['labels'] if entry['metadata']['labels']
```

Comment:
The reason I didn't do this initially is that it's still two lines, but I've changed it so that at least it's only one statement even if it's still two lines